### DISCUSSION TITLE
Howto change priority in Project and Project Tasks?

### DISCUSSION THREAD

**Discussion by antcorro** [YetiForceCompany/YetiForceCRM#15490] [How to] [answered]
Hello,

the field Priority in Project and Project task is readonly (YetiForce 6.2.0 service pack 4).
How can I change that value?

Thank you

Regards
Antonio

**Comment by mariuszkrzaczkowski** [comment]
are you sure ?
![image](https://user-images.githubusercontent.com/10198654/144008542-c084344e-5bd1-47e2-b3aa-08a9a847fdc9.png)

**Reply by antcorro** [reply]
Hello Mariusz Krzaczkowski,
thanks for reply. Yes I'm sure. 
I mean the field "priority" in the Project details (and Project Task details).
As rskrzypczak said, the problem are the default values of the field. The values by default are all in lowercase (low, normal, high). If I change the default values with the first letter in uppercase (Low, Normal,High) the field "priority" became editable in Project details

Thanks
Antonio

**Comment by rskrzypczak** [comment, marked as answer]
You have to fix the default values.
There likely is a problem with the capitalization of characters in a dictionary value.
See: https://github.com/YetiForceCompany/YetiForceCRM/commit/62be5dcc6dfa8a525ead63c5d629c11e9f2c327c

**Reply by antcorro** [reply]
Great,
thank you. The problem is that the values are in lowercase. I just renamed the values as Low, Normal, High (the first letter in Uppercase) and now it works

Thank you
Antonio
